pilot/cmd/pilot-discovery/options.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"crypto/tls"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"istio.io/istio/pilot/pkg/bootstrap"
+)
+
+// insecureTLSCipherNames returns a list of cipher suite names implemented by crypto/tls
+// which have security issues.
+func insecureTLSCipherNames() []string {
+	cipherKeys := sets.NewString()
+	for _, cipher := range tls.InsecureCipherSuites() {
+		cipherKeys.Insert(cipher.Name)
+	}
+	return cipherKeys.List()
+}
+
+// secureTLSCipherNames returns a list of cipher suite names implemented by crypto/tls.
```

Comment:
Based on the function name, the comment "secureTLSCipherNames returns a list of cipher suite names implemented by crypto/tls" should be "secureTLSCipherNames returns a list of secure cipher suites implemented by crypto/tls" instead.